Backend/router/v1/user.js

```diff
@@ -13,15 +13,36 @@ router.all('/', (req, res) => {
 });
 
 router.post('/signup', (req, res) => {
+
   let hashPassword = crypto.createHash("sha512").update(req.body.password).digest("hex");
 
-  models.user.create({
-    email: req.body.email,
-    password: hashPassword,
+  models.user.findAll({
+    where: {
+      email: req.body.email
+    }
+
   }).then(result => {
-    console.log("everything is good");
-    res.send(result);
+    if (result==""){
+
+      models.user.create({
+        email : req.body.email,
+        password : hashPassword,
+      }).then(result => {
+          console.log("< " + req.body.email + " > membership has been completed.");
```

Comment:
Do you know where this console.log it getting printed?
Like @shberry pointed, it is not a good idea to use console.log for production.

Reply:
Sorry, it was my mistake. I deleted console but it seems to have been upload on github before the code was revised.